bobcat/bb-aem-touch-ui/src/main/java/com/cognifide/qa/bb/aem/touch/pageobjects/touchui/ConfigDialog.java

```diff
@@ -99,6 +100,21 @@ public void configureWith(ComponentConfiguration config) {
     confirm();
   }
 
+  public DialogField getFieldOnTab(String label, String tab, Class fieldType) {
+    switchTab(tab);
+    return getField(label, fieldType);
+  }
+
+  public DialogField getField(String label, Class fieldType) {
+    WebElement parent;
+    if (tabs.isEmpty()) {
```

Comment:
Logic or naming of our fields seems weird in this case - 'if there are no tabs, take the active one'?
Also:
```
WebElement parent = tabs.isEmpty() ? dialog : activeTab;
```